dxr/static/css/dxr.css

```diff
@@ -198,6 +195,31 @@ mark {
     background-color: transparent;
 }
 
+/* promoted stuff */
```

Comment:
Have you played with moving the path results into a right-hand sidebar? There was that violent reaction against using vertical real estate for them at the Whistler session.